KrampitzKreutzwiserSettlersOfCatan/src/textures/ImageRef.java

```diff
@@ -5,10 +5,18 @@
  */
 package textures;
 
+import java.awt.Image;
+import javax.swing.ImageIcon;
+

```

Comment:
Prepared this mainly to hold all the Image objects but ran into some issues after the merge. I'll get back to this later.